### ISSUE TITLE
Maybe some delay and not send ajax on every key press?

### ISSUE THREAD

**Issue by nleo** [autocompletejs/autocomplete.js#11]
Wait until user stop typing. Like here http://stackoverflow.com/a/7849308/4559221

**Comment by nleo** [comment]
```javascript
clearTimeout(window.autocompleteTimer);
window.autocompleteTimer = setTimeout(function() {
    request = ajax(request, custParams, custParams.paramName + "=" + inputValue, input, result);
}, 1000);
```

**Comment by baptistedonaux** [comment]
This feature can be developed but add a low gain because, when a key is press, the last request is aborted to limit the number of current request at 1 maximum.

You can send a pull request by suggesting a parameter to customize the time to wait.

**Comment by baptistedonaux** [comment]
Feature implemented in ae4c29c4c5548f18c088ef4ba41e34d7c4560472 . Thank you for your feedback and your participation. Come back if you think can help in autocomplete.js development.
